core/chains/evm/logpoller/log_poller.go

```diff
@@ -319,15 +353,15 @@ func (lp *logPoller) getCurrentBlockMaybeHandleReorg(ctx context.Context, curren
 		err2 = lp.orm.q.WithOpts(pg.WithParentCtx(ctx)).Transaction(func(tx pg.Queryer) error {
 			// These deletes are bounded by reorg depth, so they are
 			// fast and should not slow down the log readers.
-			err2 = lp.orm.DeleteRangeBlocks(blockAfterLCA.Number.Int64(), currentBlockNumber, pg.WithQueryer(tx))
-			if err2 != nil {
-				lp.lggr.Warnw("Unable to clear reorged blocks, retrying", "err", err2)
-				return err2
+			err3 := lp.orm.DeleteRangeBlocks(blockAfterLCA.Number.Int64(), currentBlockNumber, pg.WithQueryer(tx))
```

Comment:
This change is only done for readability? Assigning to err2 would be perfectly fine since we return the error anyway so any error inside will be assigned to err2 after the return. It would make sense in a case were we don't return the error. I'm also fine with it if it's simply for readability.

Reply:
yeah just for readability